### ISSUE TITLE
Migration Action throws error when there are non Action files within the actions folder

### ISSUE THREAD

**Issue by milewski** [TheDragonCode/laravel-migration-actions#100]
### Environment

- PHP Version: 8.1
- Database Driver & Version: MySQL
- Migration Actions Version: 3.x
- Laravel Version: 9.x

### Issue description

I have a subfolder within my actions folder that does not contain Action classes.. example:

<img width="609" alt="image" src="https://user-images.githubusercontent.com/2874967/198516641-10db7107-255d-4f42-a035-a971d48c5f2a.png">

By Running `php artisan migrate:actions` I get the following error:

```php
  DragonCode\LaravelActions\Services\Migrator::resolveAction(): Return value must be of type DragonCode\LaravelActions\Action, int returned
```

I think files that do not extend `Action` shouldn't be taken into account or files that don't start with `xxxx_xx_xx_xxxxx_.....` etc...

### Steps to reproduce

Create any file within a subfolder inside the actions directory and run `php artisan migrate:actions`

**Comment by andrey-helldar** [comment]
Yes, in the 3rd version all subfolders are now read.

You gave me an idea: today I will make it possible to set exceptions for folders when searching for files in the settings.

**Comment by milewski** [comment]
Wouldn't it be easier to have a regular expression excluding any file that doesn't match the pattern? `xxxx_xx_xx_xxxxx`?

**Comment by andrey-helldar** [comment]
I thought about it, but, unfortunately, not all developers prefer this file format. I have seen projects where files were created manually without a time prefix.

Making an exception for a folder is easy, I will write tests longer :-)

As soon as I get to the computer, I'll do it.

**Comment by andrey-helldar** [comment]
Released in [`v3.1.0`](https://github.com/TheDragonCode/laravel-migration-actions/releases/tag/v3.1.0).

Run the `composer update` console command and update your [`config file`](https://github.com/TheDragonCode/laravel-migration-actions/blob/main/config/actions.php#L40-L59).
